src/hotspot/share/opto/escape.cpp

```diff
@@ -2819,14 +2819,14 @@ int ConnectionGraph::find_init_values_null(JavaObjectNode* pta, PhaseValues* pha
         offsets_worklist.append(offset);
         Node* value = nullptr;
         if (ini != nullptr) {
-          // StoreP::memory_type() == T_ADDRESS
+          //StoreP::value_basic_type() == T_ADDRESS
```

Comment:
```suggestion
          // StoreP::value_basic_type() == T_ADDRESS
```
Nit: generally we have a space after the comment.